.circleci/config.yml

```diff
@@ -96,12 +96,18 @@ jobs:
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
-          name: Get wiremock # match version in docker-compose-test.yml
-          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/2.35.0/wiremock-jre8-standalone-2.35.0.jar
+          name: Get wiremock (latest version)
+          command: |
+            LATEST_WIREMOCK_VERSION=$(curl --silent https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/maven-metadata.xml | sed -n 's/[[:space:]]*<latest>\(.*\)<\/latest>/\1/p')
+            echo "Wiremock latest version is: $LATEST_WIREMOCK_VERSION"
+            curl -o wiremock.jar "https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/$LATEST_WIREMOCK_VERSION/wiremock-standalone-$LATEST_WIREMOCK_VERSION.jar"
       - run:
           name: Run wiremock
           command: java -jar wiremock.jar --port 9091
           background: true
+      - run:
+          name: Wait for wiremock to start
```

Comment:
Because the OIDC Discovery Endpoint needs to be mocked _before_ starting the node app a delay is needed here to make sure Wiremock is available.